pkgs/tools/networking/ipfetch/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenvNoCC, fetchFromGitHub, bash, makeWrapper, coreutils, wget, gnused }:
+
+let
+  depsPath = lib.makeBinPath [
+    coreutils
+    wget
+    gnused
+  ];
+in stdenvNoCC.mkDerivation rec {
+  pname = "ipfetch";
+  version = "unstable-2022-03-14";
+
+  src = fetchFromGitHub {
+    owner = "trakBan";
+    repo = "${pname}";
+    rev = "ef082d51ee2c34d0fea7dae09126f544070c269d";
+    sha256 = "sha256-PZjFF/K7bvPIjGVoGqaoR8pWE6Di/qJVKFNcIz7G8xE=";
+  };
+
+  strictDeps = true;
+  buildInputs = [ bash ];
+  nativeBuildInputs = [ makeWrapper ];
+  postPatch = ''
+    patchShebangs --host ${pname}

```

Comment:
It's probably better to not be so liberal with pname usage. If anyone override the package name (for whatever reason), it would break this derivation. Since the source will generally still use the ipfetch name for folders and build outputs.